build_taglib.bat

```diff
@@ -32,50 +32,25 @@ mkdir %PLATFORM%
 cd %PLATFORM%
 REM NOTE(pegasus): generated solution with
 
-"%CMAKEDIR%\cmake" ../ -G %COMPILERPROJECT% -DBUILD_SHARED_LIBS=%DYNAMIC% -DCMAKE_SUPPRESS_REGENERATION=1 -DCMAKE_USE_RELATIVE_PATHS=1 -DZLIB_INCLUDE_DIR=%INCLUDE_DIR% -DZLIB_LIBRARY=%LIB_DIR%\%ZLIBNAME%
+"%CMAKEDIR%\cmake" .. -G %COMPILERPROJECT% -DBUILD_SHARED_LIBS=%DYNAMIC% -DCMAKE_SUPPRESS_REGENERATION=1 -DZLIB_INCLUDE_DIR=%INCLUDE_DIR% -DZLIB_LIBRARY=%LIB_DIR%\%ZLIBNAME% -DCMAKE_PREFIX_PATH=%ROOT_DIR%\Qt-%QT_VERSION%\lib\cmake -DCMAKE_INSTALL_PREFIX=%ROOT_DIR% -DCMAKE_INSTALL_INCLUDEDIR=%INCLUDE_DIR%
```

Comment:
Could you edit `build_environment.bat` to set a `PLATFORM_TOOLSET` environment variable to `v%VCVERSION%_xp` and then set it via `-T %PLATFORM_TOOLSET%` here?

Reply:
Ah yes, thats it.